tools/pipelines-tasks/common/nuget_helper.ts

```diff
@@ -0,0 +1,72 @@
+import * as path from "path";
+import * as tl from "azure-pipelines-task-lib/task";
+import * as fs from 'fs';
+import fetch from 'node-fetch';
+
+export async function downloadNugetTool(url: string, downloadLoc: string): Promise<void> {
```

Comment:
Adding JSDoc comments to the functions could improve readability and maintainability

Reply:
Done